packages/@o3r/schematics/src/utility/collection.spec.ts

```diff
@@ -43,18 +42,18 @@ const angularJsonNoGeneric: WorkspaceSchema = {
   } as any
 };
 
-describe('Get schematics options', () => {
-  const createFakeContext = (collection: string, name: string): any => ({
-    schematic: {
-      description: {
-        collection: {
-          name: collection
-        },
-        name
-      }
+const createFakeContext = (collection: string, name: string): any => ({
```

Comment:
wasn't this rule disabled for tests? (I am ok with the change, just curious)

Reply:
We said that we will deactivate this rule after the migration, for now we try to fix a maximum when possible and meaningful